flict/__main__.py

```diff
@@ -81,7 +81,7 @@ def parse():
     description = "NAME\n  " + PROGRAM_NAME + "\n\n"
     description = description + "DESCRIPTION\n  " + PROGRAM_DESCRIPTION + "\n\n"
 
-    epilog = ""
+    epilog = "Return codes:\n{codes}\n\n".format(codes=ReturnCodes.get_help())
```

Comment:
This one needs a refresher after https://github.com/vinland-technology/flict/pull/101

Reply:
Done!